packages/bodiless-page/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@bodiless/page",
+  "version": "0.3.6",
+  "description": "BodilessJS page operations",
+  "author": "Guilherme Zeni <gzeni@its.jnj.com>",
+  "license": "Apache-2.0",
+  "sideEffects": false,
+  "directories": {
+    "test": "__tests__"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/johnsonandjohnson/bodiless-js.git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "build": "tsc -p ./tsconfig.json && npm run build:api-doc",
+    "build:watch": "npm run build -- --watch",
+    "build:api-doc": "typedoc --out doc/api src",
+    "lint": "tslint ./src/**/*.ts?x",
+    "clean": "rimraf \"lib/*\" && rimraf tsconfig.tsbuildinfo && rimraf \"doc/api\""
+  },
+  "dependencies": {
+    "@bodiless/components": "^0.3.6",
+    "@bodiless/core": "^0.3.6",
+    "@bodiless/fclasses": "^0.3.6",
+    "@bodiless/ui": "^0.3.6",
+    "@bodiless/gatsby-theme-bodiless": "^0.3.6",
```

Comment:
@bodiless/page has dependency on gatsby-theme-bodiless? I thought gatsby-theme-bodiless depends on @bodiless/page

Reply:
Hello @dewen ,

That's true!
I discussed with Chris and I provided a short term workaround to remove that dependency right now. Please, check it again. Thanks!